hack/run-functests.sh

```diff
@@ -62,7 +62,7 @@ VALIDATION_SUIT_SUBSTR="validation"
 
 go version
 if [[ $SUITE == *"$VALIDATION_SUIT_SUBSTR"* ]]; then
-  GOFLAGS=-mod=vendor ginkgo --output-dir=$JUNIT_OUTPUT_DIR --junit-report=$JUNIT_OUTPUT_FILE -v -p "$SUITE"
+  GOFLAGS=-mod=vendor ginkgo --output-dir=$JUNIT_OUTPUT_DIR --junit-report=$JUNIT_OUTPUT_FILE -vv -p "$SUITE"
```

Comment:
Perhaps we can add an envvar for the verbosity? Would make it easier for when we're doing stuff manually to debug.